### ISSUE TITLE
Remove console logs

### ISSUE THREAD

**Issue by nicolasrouanne** [Netflix/restful-jsonapi#16]
Is it possible to remove [outputs](https://github.com/Netflix/restful-jsonapi/blob/bc44c502a3cc76a5eab9a645c3c2b3dfcad61b1e/lib/restful/jsonapi/railtie.rb#L6) to console? 

```
Extending ActionController::Base with Restful::Jsonapi::RestifyParam
Extending ActionController::Base with Restful::Jsonapi::SerializableErrors
Extending ActionController::API with Restful::Jsonapi::RestifyParam
Extending ActionController::API with Restful::Jsonapi::SerializableErrors
```

Or make them run under some environment only (such as `development`)? It's just that having some printouts feels a little clumsy, and I don't really see the use for it...

**Comment by nicolasrouanne** [comment]
Hum I've seen it's addressed by an open PR #13 ... Any chance it can be merged?

**Comment by nicolasrouanne** [comment]
Closing since #13 is merged! 🎉
